src/models/UserMapper.php

```diff
@@ -371,4 +371,21 @@ public function getUserIdFromEmail($email) {
         }
         return false;
     }
+
+    /**
+     * Used only on test platforms, if the config is enabled
+     * 
+     * Designed to allow creation of verified users for testing purposes
+     */
+    public function verifyThisTestUser($user_id) {
```

Comment:
Ideally, I think that this method should be called verifyThisUser($user_id) and then used within verifyUser(), so that we don't duplicate this code.

If we want to have the word "test" in the UserMapper's public api, then verifyThisUser() could be protected/private and verifyThisTestUser() could just call it,